canonical-web-and-design/charmhub.io
# Move link to charm page from listing

## Done
Moved the link and save actions from bottom of listing page to top and updated the logic so it behaved more consistantly

## QA
- Go to https://charmhub-io-1197.demos.haus/landscape-keystone/listing
- Check that the `Updates to this information will appear immediately...` message is at the top of the page and that the buttons are disabled
- Check if you make changes to the form the buttons are enabled
- Check if you revert the form the buttons are disabled again

## Issue
Fixes #1148
